src/blocks/header/lotus_json.rs

```diff
@@ -0,0 +1,162 @@
+// Copyright 2019-2023 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+use crate::lotus_json::*;
+use once_cell::sync::OnceCell;
+use serde::{Deserialize, Serialize};
+
+use super::BlockHeader;
+
+#[derive(Serialize, Deserialize)]
+#[serde(rename_all = "PascalCase")]
+pub struct BlockHeaderLotusJson {
+    miner: AddressLotusJson,
```

Comment:
In general, why do you put `LotusJson` everywhere? Is it so distinct from a regular `Json`?

Reply:
This is a piecewise upgrade of our codebase - I don't want there to be two different `ActorStateJson`s.
I could rename to `FilJson` if this is a blocker, but going to leave for now 🤷